charts/r2devops/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: r2devops
 description: Helm chart for R2Devops
 type: application
-version: 0.3.8
-appVersion: "1.16.1"
+version: 0.4.0
+appVersion: "1.16.0"
```

Comment:
Is it normal to lower the appVersion?

Reply:
Nope :) it was a mistake during a rebase